query10.sql

```diff
@@ -1,7 +1,12 @@
 /*
   Using the station status dataset, find the distance in meters of all stations
   from Meyerson Hall.
+
+  Too many to list, but see query below
+  
 */
 
 -- Enter your SQL query here
-select ...
+
+SELECT name, (the_geom <-> ST_SETSRID(ST_MAKEPOINT(-75.1923, 39.9520), 4326)::geometry) * 111139 as distance_to_meyerson
```

Comment:
Should be transforming the geometry to some SRID other than 4326 (e.g. 2272 or 32129)